files/edge-core/scripts/arm_update_active_details.sh

```diff
@@ -26,4 +26,6 @@
 . $(dirname "$0")/arm_update_cmdline.sh
 
 # copy header if present
-[ -n "$ACTIVE_HDR" ] && cp "$ACTIVE_HDR" "$HEADER"
+[ -f "$ACTIVE_HDR" ] && cp "$ACTIVE_HDR" "$HEADER"
+
+exit 0
```

Comment:
We have two scenarios here where `exit` may not be desirable.
1. the last `cp` returns an error which is discarded by `exit`
2. the header file doesn't exist, but we want it to always exist (is this the case?), and the "error" from `[ -f ]` will be hidden by `exit`.
(If the existence of the header isn't mandatory, we can shape the `[ -f ]` line so that doesn't cause an error.)

Reply:
The real fix is the change to -f.  The cp  was failing because ACTIVE_HDR was defined, but the referenced file didn't exist, thus causing the cp to  fail which shows errors in the startup log, but those errors are safely handled by edge-core.  So, it's ok if the header doesn't exist (at least to clear up errors for this PR, we might want to investigate edge-core further)

Reply:
We can use this as is. An approach to ignore `[ -f ]` errors but not cp errors can be using `if` (which doesn't set an error code if the condition is false).